docs/deployment/settings.md

```diff
@@ -301,6 +301,7 @@ kyuubi.frontend.protocols|THRIFT_BINARY|A comma separated list for all frontend
 kyuubi.frontend.proxy.http.client.ip.header|X-Real-IP|The http header to record the real client ip address. If your server is behind a load balancer or other proxy, the server will see this load balancer or proxy IP address as the client IP address, to get around this common issue, most load balancers or proxies offer the ability to record the real remote IP address in an HTTP header that will be added to the request for other devices to use. Note that, because the header value can be specified to any ip address, so it will not be used for authentication.|string|1.6.0
 kyuubi.frontend.rest.bind.host|&lt;undefined&gt;|Hostname or IP of the machine on which to run the REST frontend service.|string|1.4.0
 kyuubi.frontend.rest.bind.port|10099|Port of the machine on which to run the REST frontend service.|int|1.4.0
+kyuubi.frontend.rest.max.worker.threads|999|Maximum number of threads in the of frontend worker thread pool for the rest frontend service|int|1.6.2
```

Comment:
the default value changes 200 -> 999, will this cause problems

Reply:
<img width="962" alt="image" src="https://user-images.githubusercontent.com/6757692/206622122-1e53b0b1-4ef6-4f03-97fa-a99687e9d25b.png">
It should be fine, the min thread is 8.